applications/peripheral_sensor_node/README.rst

```diff
@@ -0,0 +1,264 @@
+The application holds a number of measurements depending on the size of memory allocated for storage.
+New data automatically replaces oldest records.
+
+Application Overview
+********************
+
+To perform its tasks, the nRF Peripheral Sensor node application uses components available in Zephyr and the |NCS|, namely the :ref:`lib_caf` modules and :ref:`zephyr:sensor_api` for sampling sensors, :ref:`nsms_readme` for presenting measured data, and a dedicated service for historical data retrieval over Bluetooth LE.
+The application also stores measurements in a dedicated non-volatile memory partition.
+You can download the stored data over Bluetooth.
+
+Sampling sensors
+================
+
+The application handles the sensor sampling using the Zephyr's :ref:`zephyr:sensor_api`.
+This approach allows to use any sensor available in Zephyr.
+
+By default, you need to use the :ref:`peripheral_sensor_node_shield` for this application.
+
+Power management
+================
+
+To reduce power consumption, the Peripheral Sensor node application mostly remains in the power-down state.
+The application periodically performs burst measurement of the environmental and IMU sensors in specified intervals and does additional burst measurement when triggered by the interrupt from the ``ADXL362`` sensor.
+You can configure the periodic burst measurement intervals through the :ref:`CONFIG_APP_CONTROL_BURST_INTERVAL_S <CONFIG_APP_CONTROL_BURST_INTERVAL_S>` Kconfig option.
+By default, the burst interval is set to 120 seconds.
+You can configure the number of measurements for a single burst using the :ref:`CONFIG_APP_CONTROL_BURST_MEAS_CNT <CONFIG_APP_CONTROL_BURST_MEAS_CNT>` Kconfig option, and the time interval between measurements is specified in the :ref:`CONFIG_APP_CONTROL_BURST_MEAS_INTERVAL_MS <CONFIG_APP_CONTROL_BURST_MEAS_INTERVAL_MS>` Kconfig option.
+
+When the application performs burst measurements, it also starts advertising.
+This means it can be connected through a Bluetooth central device, for example, the `nRF Connect for Mobile`_ application on your smartphone or tablet.
+
+Firmware architecture
+=====================
+
+The nRF Peripheral Sensor node application has a modular structure, where each module has a defined scope of responsibilities.
+The application uses the :ref:`app_event_manager` to distribute events between modules in the system.
+
+The following figure shows the application architecture and visualizes relations between Application Event Manager, modules, drivers, and libraries.
+
+.. figure:: /images/peripheral_sensor_node_arch.svg
+   :alt: Peripheral Sensor node application architecture
+
+   Peripheral Sensor node application architecture
+
+Requirements
+************
+
+The application supports the following development kit:
+
+.. table-from-sample-yaml::
+
+User interface
+**************
+The application uses one LED to display the application state.
+``LED 1`` lit when the application is active
```

Comment:
```suggestion

LED 1:
   The LED is lit when the application is active.
```
Just to make sure - is 'LED 1' the updated numbering used on newer PDKs?

Reply:
Application uses LED1.